stories/components/SInputNumber.02_Addons.story.vue

```diff
@@ -0,0 +1,286 @@
+            </template>
+          </SInputNumber>
+
+          <div class="actions">
+            <SButton size="small" mode="info" label="Submit" @click="validateAndNotify" />
+            <SButton size="small" mode="mute" label="Reset" @click="init" />
+          </div>
+        </div>
+      </div>
+      <div class="group">
+        <div class="title">Addon Button (After / Icon)</div>
+        <div class="grid">
+          <SInputNumber
+            name="lottery2"
+            label="Label"
+            info="Some helpful information."
+            note="Required"
+            help="Please fill in 7-digit numbers"
+            placeholder="1000000"
+            :validation="validation.lottery2"
+            v-model="data.lottery2"
+          >
+            <template #addon-after>
+              <SInputAddon :label="IconLightbulb" @click="data.lottery2 = generate7DigitNumbers()" />
+            </template>
+          </SInputNumber>
+
+          <div class="actions">
+            <SButton size="small" mode="info" label="Submit" @click="validateAndNotify" />
+            <SButton size="small" mode="mute" label="Reset" @click="init" />
+          </div>
+        </div>
+      </div>
+      <div class="group">
+        <div class="title">Addon Dropdown (Before / Single Select Dropdown)</div>
+        <div class="grid">
+          <SInputNumber
+            name="currency"
+            label="Label"
+            info="Some helpful information."
+            note="Required"
+            help="Please fill in currency and amount."
+            placeholder="1000000"
+            :validation="validation.currency"
+            v-model="data.currency"
+            separator
+          >
+            <template #addon-before>
+              <SInputAddon :dropdown="dropdown1" />
+            </template>
+          </SInputNumber>
+
+          <div class="actions">
+            <SButton size="small" mode="info" label="Submit" @click="validateAndNotify" />
+            <SButton size="small" mode="mute" label="Reset" @click="init" />
+          </div>
+        </div>
+      </div>
+      <div class="group">
+        <div class="title">Addon Dropdown (After / Dropdown Menu)</div>
+        <div class="grid">
+          <SInputNumber
+            name="amount3"
+            label="Label"
+            info="Some helpful information."
+            note="Required"
+            help="Please fill in amount."
+            placeholder="1000000"
+            :validation="validation.amount3"
+            v-model="data.amount3"
+            separator
+          >
+            <template #addon-after>
+              <SInputAddon :label="IconDotsThree" :dropdown="dropdown2" no-dropdown-arrow />
+            </template>
+          </SInputNumber>
+        </div>
+      </div>
+      <div class="group">
+        <div class="title">Addon Dropdown (Before / After / Disabled)</div>
+        <div class="grid">
+          <SInputText
```

Comment:
This is a typo for `SInputNumber`, and so are the props.

Reply:
Fixed!